.github/workflows/build_documentation.yml

```diff
@@ -26,7 +27,7 @@ jobs:
         run: |
           sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev
 
-          pip install -e .
+          pip install git+https://${{ secrets.HUGGINGFACE_PUSH }}@github.com/huggingface/doc-builder
```

Comment:
Why do you need the secret? The repo is public.

Reply:
Oh right :)

Reply:
Will update once I validate that it's passing here